### PR TITLE
Unit Tests - Vertical bar chart

### PR DESCRIPTION
**Pull request by srmukher**
Added unit tests for vertical bar chart. Improved coverage from **77.04% to 91.42%**

Before:
image

After:
image

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/f94b6d05d7642563cb96e2718402fbbbab65cbbc'>f94b6d05d7642563cb96e2718402fbbbab65cbbc</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: packages/react-charting/src/components/VerticalBarChart/VerticalBarChartRTL.test.tsx [hunk 1]
```diff
@@ -465,3 +686,187 @@ describe('Vertical bar chart re-rendering', () => {
     });
   });
 });
+
+describe('VerticalBarChart unit tests', () => {
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/VerticalBarChart/VerticalBarChartRTL.test.tsx:690]: Lets move this to a UT file

File: packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx [hunk 2]
```diff
@@ -191,6 +191,42 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
     );
   }
 
+  public getDomainMargins() {
```
Comment (AtishayMsft) [inline packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx:194]: Are these functions used anywhere in the product code.

Comment (srmukher) [reply packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx:194]: no, added public handlers for testing of the private functions

File: packages/react-charting/src/components/VerticalBarChart/VerticalBarChartRTL.test.tsx (hunk 1 above)
Comment (srmukher) [reply packages/react-charting/src/components/VerticalBarChart/VerticalBarChartRTL.test.tsx]: done

**Comment by AtishayMsft** [comment]
@srmukher Could you close these old PRs which are not needed anymore

**Comment by srmukher** [comment]
Closing this PR as the unit tests have been checked in via another PR.

**Pull request closed without merging**